### ISSUE TITLE
first commit

### ISSUE THREAD

**Issue by Usiniverse** [jangkyoungtae/pracproject#1]
<!doctype html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
          integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
crossorigin="anonymous">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
            integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
            crossorigin="anonymous"></script>
    <title>스파르타코딩클럽 | 부트스트랩 연습하기</title>
    <style>
        .mytitle {
            background-color: green;
            height: 250px;
            width: 100%;
            background-image: linear-gradient(0deg, rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("https://search.pstatic.net/common/?src=http%3A%2F%2Fcafefiles.naver.net%2FMjAxODA1MjFfMjc0%2FMDAxNTI2ODYzMDgxMTkx.Mv_xIzMHQVzrtfYzPnYV7lGdJaGSeFpfIwCGpIugzjog.mxuWtvHCm-8b027NGMznVNdf7wcnuXWcSLDWrYNQTTEg.JPEG.andy7144%2FexternalFile.jpg&type=sc960_832");
            background-position: center 35%;
            background-size: cover;
            color: white;
            display: flex;
            flex-direction: column;
            justify-content: center;
            align-items: center;
        }

        .mytitle > button {
            width: 200px;
            height: 50px;

            background-color: transparent;
            color: white;

            border-radius: 50px;

            border: 1px solid white;

            margin-top: 10px;
        }

        .mypost {
            width: 500px;
            margin: 20px auto 0 auto;
            box-shadow: 0px 0px 3px 0px gray;
            padding: 20px;
        }

        .mybtns {
            margin-top: 20px;
        }

        .card {
            width:500px;
            margin:10px auto 0 auto;
            box-shadow: 0px 0px 1px 0px gray;
        }

    </style>
    <script>
        $(document).ready(function () {
            alert('다 로딩됐다!')
        });
        $.ajax({
                type: "GET",
                url: "http://spartacodingclub.shop/sparta_api/weather/seoul",
                data: {},
                success: function (response) {
                    $('#temp').text(response['temp'])
                    }
            })
    </script>
</head>
<body>
    <div class="mytitle">
        <h1>I'm a big fan of Andres Iniesta!</h1>
        <p>현재 기온 : <span id="temp">00.00도</span></p>
    </div>
    <div class="mypost">
        <div class="form-floating mb-3">
            <input type="email" class="form-control" id="floatingInput" placeholder="name@example.com">
            <label for="floatingInput">영화URL</label>
        </div>
        <div class="form-floating">
            <textarea class="form-control" placeholder="Leave a comment here" id="floatingTextarea"></textarea>
            <label for="floatingTextarea">코멘트</label>
        </div>
        <div class="mybtns">
            <button type="button" class="btn btn-dark">응원남기기</button>
        </div>
    </div>
    <div class="card">
        <div class="card-body">
            <blockquote class="blockquote mb-0">
                <p>당신의 플레이는 최고입니다!</p>
                <footer class="blockquote-footer"><cite title="Source Title">호빵맨</cite>
                </footer>
            </blockquote>
        </div>
    </div>
    <div class="card">
        <div class="card-body">
            <blockquote class="blockquote mb-0">
                <p>당신의 플레이는 최고입니다!</p>
                <p>최고최고!</p>
                <footer class="blockquote-footer"><cite title="Source Title">호빵맨</cite>
                </footer>
            </blockquote>
        </div>
    </div>
    <div class="card">
        <div class="card-body">
            <blockquote class="blockquote mb-0">
                <p>당신의 플레이는 최고입니다!</p>
                <p>최고최고!</p>
                <footer class="blockquote-footer"><cite title="Source Title">호빵맨</cite>
                </footer>
            </blockquote>
        </div>
    </div>
</body>

</html>
